lddtree.sh

```diff
@@ -96,11 +97,14 @@ elf_specs_binutils() {
 		| tr '\n' ' '
 }
 
+
+
+
 # elf wrapper functions
-elf_rpath() { elf_rpath_$BACKEND "$@"; }
-elf_interp() { elf_interp_$BACKEND "$@"; }
-elf_needed() { elf_needed_$BACKEND "$@"; }
-elf_specs() { elf_specs_$BACKEND "$1"; }
+elf_rpath() { [ ! -z "$1" ] && elf_rpath_$BACKEND "$@" | sed -e "s:[$]ORIGIN:${1%/*}:g" | sed -e "s:[$]{ORIGIN}:${1%/*}:g" | sed -e "s,:\.,:${1%/*},g" | sed -e "s,^\.,${1%/*},g"; }
```

Comment:
- What is this supposed to solve?
- Why do you spawn 4 `sed -e` instead of 1 `sed` with 4 `-e` parameters?